Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -794,16 +794,17 @@ BeaconAPIEnabled:
 
 BlobRegistryTopOriginPartitioningEnabled:
   type: bool
-  status: unstable
+  status: testable
+  category: dom
   humanReadableName: "Partition Blob URL Registry"
   humanReadableDescription: "Partition Blob URL Registry by Top-Level Origin"
   defaultValue:
     WebKitLegacy:
-      default: false
+      default: true
```

Comment:
You marked it as testable but then enabled it everywhere. This doesn't sound right. Either it should be `testable` or `preview` and remain false by default. Or we should mark it as `stable` if you want to enable the feature by default.